fluss-server/src/test/java/com/alibaba/fluss/server/zk/ZooKeeperClientTest.java

```diff
@@ -220,6 +220,31 @@ void testBatchCreateLeaderAndIsr() throws Exception {
         }
     }
 
+    @Test
+    void testBatchUpdateLeaderAndIsr() throws Exception {
+        int totalCount = 100;
+
+        // try to register bucket leaderAndIsr
+        Map<TableBucket, LeaderAndIsr> leaderAndIsrList = new HashMap<>();
+        for (int i = 0; i < totalCount; i++) {
+            TableBucket tableBucket = new TableBucket(1, i);
+            LeaderAndIsr leaderAndIsr =
+                    new LeaderAndIsr(i, 10, Arrays.asList(i + 1, i + 2, i + 3), 100, 1000);
+            leaderAndIsrList.put(tableBucket, leaderAndIsr);
+            zookeeperClient.registerLeaderAndIsr(tableBucket, leaderAndIsr);
+        }
```

Comment:
Is it necessary to call `register leaderAndIsr`? I'm a bit confused about this code because this makes even if `batchUpdateLeaderAndIsr` fails, the subsequent assertion on `leaderAndIsr` could still pass. For example, when I remove line 238 (`zookeeperClient.batchUpdateLeaderAndIsr(leaderAndIsrList)`), the test still passes. Maybe we need to remove this part code.

Reply:
Thank you for pointing out the issue here. However, we do need to call `register leaderAndIsr` first, otherwise there will be a "node does not exist" problem when updating. In the previous implementation, I didn't use the new `LeaderAndIsr` to test `batchUpdateLeaderAndIsr`, which resulted in the test passing even after deletion (`zookeeperClient.batchUpdateLeaderAndIsr(updateLeaderAndIsrList);`). Now I am using a new `LeaderAndIsr` to verify that `batchUpdateLeaderAndIsr` is indeed updated successfully.